src/infill.cpp

```diff
@@ -93,9 +114,94 @@ void Infill::generate(Polygons& result_polygons, Polygons& result_lines, const S
     }
 }
 
+void Infill::_generate(Polygons& result_lines, const SierpinskiFillProvider* cross_fill_provider, const SliceMeshStorage* mesh)
```

Comment:
Why did you choose to move this part to a private function?

Reply:
The original _generate( ... ) which I intend to overload was also private. Not sure if I'm gonna need this part in the final implementation.